IObundle/iob-soc-opencryptohw
# Milestone 7 part 2 Accelerator Synthesis; Milestone 10 part 2: System Synthesis

- update with main branch
- Milestone 7 part 2: Accelerator Synthesis
  - test with the command:
  ```bash
  make fpga-build-versat
  ```
  - check `hardware/fpga/vivado/AES-KU040-DB-G/*.edif` files
- Milestone 10 par 2: Complete System Synthesis
  - test with the command:
  ```bash
  make fpga-build
  ```
  - check `hardware/fpga/vivado/AES-KU040-DB-G/*.log` files
- update CI to test new supported targets
